.github/ISSUE_TEMPLATE.md

```diff
@@ -35,21 +31,16 @@
 
 
 
----
+<!-- ⚠️ Make sure to browse the opened and closed issues before submitting your issue. -->
 
-*FEATURE REQUEST TEMPLATE*  <!-- Delete this header from your issue. -->
 
-**What is the expected behavior?**
 
+<!-- FEATURE REQUEST TEMPLATE -->
 
+<!-- ⚠️ If you do not respect this template your issue will be closed. -->
 
----
+**What is the expected behavior?**
```

Comment:
Maybe you prefer a title:

```
### What is the expected behavior?
```

### What is the expected behavior?

---

Example at https://raw.githubusercontent.com/QISKit/qiskit-js/master/.github/ISSUE_TEMPLATE/BUG_REPORT.md